Makefile

```diff
@@ -5,13 +5,13 @@ cs_dry_run: ## Display CS violations without fixing it
 	./vendor/bin/php-cs-fixer fix --verbose --dry-run
 
 spec_diff: ## detect diff between spec and SDK
-	CI_TEST=1 vendor/bin/jane-openapi generate --config-file=.jane-openapi.php && diff generated_ci generated
+	CI_TEST=1 vendor/bin/jane-openapi generate --config-file=.jane-openapi.php && vendor/bin/php-cs-fixer fix && diff generated_ci generated
 
 test: ## Run test suite
 	./vendor/bin/simple-phpunit
 
 jane: ## Generate the SDK
-	rm -rf generated/* && vendor/bin/jane-openapi generate --config-file=.jane-openapi.php
+	rm -rf generated/* && vendor/bin/jane-openapi generate --config-file=.jane-openapi.php && vendor/bin/php-cs-fixer fix
```

Comment:
You can configure Jane to fix your code with `use-fixer` to `true` (default is `false`)
Then you can give him a custom cs-fixer configuration with `fixer-config-file` and pointing to your configuration file.